module/channel/src/Application/Command/ExportProductCommand.php

```diff
@@ -0,0 +1,106 @@
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Channel\Application\Command;
+
+use Ergonode\Channel\Domain\Entity\ChannelId;
+use Ergonode\Channel\Domain\Repository\ChannelRepositoryInterface;
+use Ergonode\EventSourcing\Infrastructure\Bus\CommandBusInterface;
+use Ergonode\Product\Domain\Entity\ProductId;
+use Ergonode\Product\Domain\Repository\ProductRepositoryInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ */
+class ExportProductCommand extends Command
+{
+    private const NAME = 'channel:export:product';
+
+    /**
+     * @var CommandBusInterface
+     */
+    private CommandBusInterface $commandBus;
+
+    /**
+     * @var ChannelRepositoryInterface
+     */
+    private ChannelRepositoryInterface $channelRepository;
+
+    /**
+     * @var ProductRepositoryInterface
+     */
+    private ProductRepositoryInterface $productRepository;
+
+    /**
+     * @param CommandBusInterface        $commandBus
+     * @param ChannelRepositoryInterface $channelRepository
+     * @param ProductRepositoryInterface $productRepository
+     */
+    public function __construct(
+        CommandBusInterface $commandBus,
+        ChannelRepositoryInterface $channelRepository,
+        ProductRepositoryInterface $productRepository
+    ) {
+        parent::__construct(static::NAME);
+
+        $this->commandBus = $commandBus;
+        $this->channelRepository = $channelRepository;
+        $this->productRepository = $productRepository;
+    }
+
+
+    /**
+     * Command configuration
+     */
+    public function configure(): void
+    {
+        $this->addArgument('channel', InputArgument::REQUIRED, 'Channel ID.');
+        $this->addArgument('product', InputArgument::REQUIRED, 'Product ID.');
+    }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @throws \Exception
+     */
+    public function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $channelId = $input->getArgument('channel');
+        $productId = $input->getArgument('product');
+
+        $channelId = new ChannelId($channelId);
+        $productId = new ProductId($productId);
+
+        if (!$channel = $this->channelRepository->load($channelId)) {
+            $output->writeln('Channel not exists');
+        }
+
+        if (!$product = $this->productRepository->load($productId)) {
+            $output->writeln('Product not exists');
+        }
+
+        $output->writeln(
+            sprintf(
+                'Processing <comment>%s</comment> witch channel <comment>%s</comment>',
+                $product->getSku()->getValue(),
+                $channel->getName()
+            )
+        );
+
+        $command = new \Ergonode\Channel\Domain\Command\ExportProductChannelCommand($channelId, $productId);
```

Comment:
Why you not used in file header 
`use Ergonode\Channel\Domain\Command\ExportProductChannelCommand` ...